jobs/involuntary-dissolutions/involuntary_dissolutions.py

```diff
@@ -73,7 +77,46 @@ def shell_context():
     app.shell_context_processor(shell_context)
 
 
-def initiate_dissolution_process(app: Flask):  # pylint: disable=redefined-outer-name,too-many-locals
+def create_invountary_dissolution_filing(business_id: int):
+    """Create a filing entry to represent an involuntary dissolution filing."""
+    filing = Filing()
+    filing.business_id = business_id
+    filing.filing_type = CoreFiling.FilingTypes.DISSOLUTION
+    filing.filing_sub_type = DissolutionTypes.INVOLUNTARY
+    filing.save()
+
+    return filing
+
+
+async def put_filing_on_queue(filing_id: int, app: Flask, qsm: QueueService):
+    """Send queue message to filer to dissolve business."""
+    try:
+        subject = app.config['NATS_FILER_SUBJECT']
+        payload = {'filing': {'id': filing_id}}
+        await qsm.publish_json_to_subject(payload, subject)
+    except Exception as err:  # pylint: disable=broad-except # noqa F841;
+        # mark any failure for human review
+        capture_message(
+            f'Queue Error: Failed to place filing {filing_id} on Queue with error:{err}',
+            level='error'
+        )
+
+
+def mark_eligible_batches_completed():
+    """Mark batches completed if all of their associated batch_processings are compeleted."""
+    subquery = db.session.query(BatchProcessing.batch_id).filter(BatchProcessing.status.in_([
+        BatchProcessing.BatchProcessingStatus.COMPLETED,
+        BatchProcessing.BatchProcessingStatus.WITHDRAWN
+    ]))
+    batches = db.session.query(Batch).filter(Batch.id.in_(subquery)).all()
+
+    for batch in batches:
+        batch.status = Batch.BatchStatus.COMPLETED
+        batch.end_date = datetime.utcnow()
+        batch.save()
+
+
+def stage_1_process(app: Flask):  # pylint: disable=redefined-outer-name,too-many-locals 
     """Initiate dissolution process for new businesses that meet dissolution criteria."""
```

Comment:
Renaming to be consistent with the other stage processing functions. Also updated some of the description strings to be more specific as to what each stage does.